### DISCUSSION TITLE
Backend memory use on aarch64 - excessive?

### DISCUSSION THREAD

**Discussion by 2q2code** [Infisical/infisical#2887] [Q&A]
Infisical backend is idle most of the time. It's great for doing its job when it is called upon, but I'd like to know why it needs to use ~200MB of RAM continuously? The Pi I run it on doesn't have a great deal of RAM to spare, so I need to investigate these. All other processes around it (far more active ones!) use not even a fraction of this! My installation is via Docker. Image used is: infisical/infisical:latest-postgres, updated up to Dec 16.

**Comment by maidul98** [comment]
Hi @2q2code, is this pattern something you have noticed more recently?

**Reply by 2q2code** [reply]
As I have only recently begun using Infisical, yes. I have no earlier comparison. I have enabled cgroup memory accounting on my servers and this process was a standout from the others.

**Reply by 2q2code** [reply]
I checked the logs, and while there is nothing to suggest a reason why there would be such bloated memory use, it does have an error trying to connect to my mail server so it could be related to that somehow.

**Comment by bl4ckswordsman** [comment]
I get unusually high CPU usage on idle too, and about 860 RAM usage on an aarch64 Oracle cloud instance. With the infisical:latest image.
Logs do not show any warning or error.
